modules/swagger-codegen/src/main/resources/cpprest/api-source.mustache

```diff
@@ -53,7 +53,22 @@ pplx::task<{{#returnType}}{{{returnType}}}{{/returnType}}{{^returnType}}void{{/r
     utility::string_t responseHttpContentType;
     
     // use JSON if possible
-    if ( responseHttpContentTypes.size() == 0 || responseHttpContentTypes.find(U("application/json")) != responseHttpContentTypes.end() )
+    if ( responseHttpContentTypes.size() == 0 )
+    {
+        {{#vendorExtensions.x-codegen-response.isPrimitiveType}}
```

Comment:
Btw, the mustache template variables can be obtained via debug flags: https://github.com/swagger-api/swagger-codegen/wiki/Mustache-Template-Variables